sdk/samples/iot/paho_iot_hub_c2d_sample.c

```diff
@@ -44,9 +44,9 @@ static void disconnect_mqtt_client_from_iot_hub(void);
 
 static void parse_c2d_message(
     char* topic,
-    int topic_len,
-    const MQTTClient_message* message,
-    az_iot_hub_client_c2d_request* c2d_request);
+    int const topic_len,
```

Comment:
Primitive parameters like `int` that are passed by value (rather than ref/by pointer) don't need `const`.
```suggestion
    int topic_len,
```

Reply:
lol.  i must have been really excited about const, because I'm not sure why I did that :D